### ISSUE TITLE
lens available for syslog-ng ?

### ISSUE THREAD

**Issue by binups** [hercules-team/augeas#581]
lens available for syslog-ng ?

**Comment by lutter** [comment]
No, unfortunately, there isn't one yet. There's a related question for the syslog augeas provider: https://github.com/hercules-team/augeasproviders_syslog/issues/2

**Comment by raphink** [comment]
Arguably it shouldn't be too hard to write one. I just haven't had that need yet (nor our clients at this point).
